ignite/services/plugin/protocol.go

```diff
@@ -105,7 +105,7 @@ func (c client) ExecuteHookCleanUp(ctx context.Context, h *ExecutedHook, api Cli
 	return err
 }
 
-func (c client) startClientAPIServer(api ClientAPI) (uint32, func()) {
+func (c client) startClientAPIServer(api ClientAPI) (uint32, func() bool) {
```

Comment:
What should the caller do with the bool? Currently, it will be discarded: https://github.com/ignite/cli/blob/bc2d5d66b8beb006b243b40fbf8c9bc0c1326bb1/ignite/services/plugin/protocol.go#L104